data-prepper-plugins/otel-metrics-source/src/main/java/org/opensearch/dataprepper/plugins/source/otelmetrics/OTelMetricsGrpcService.java

```diff
@@ -51,6 +51,14 @@ public OTelMetricsGrpcService(int bufferWriteTimeoutInMillis,
         requestProcessDuration = pluginMetrics.timer(REQUEST_PROCESS_DURATION);
     }
 
+    public void rawExport(final ExportMetricsServiceRequest request) {
+        try {
+            if (buffer.isByteBuffer()) {
+                buffer.writeBytes(request.toByteArray(), null, bufferWriteTimeoutInMillis);
+            }
+        } catch (Exception e) {
+        }
```

Comment:
We need exception handling here and emit metric for failure.

Reply:
This is only used for testing.